assets/css/reset.css

```diff
@@ -0,0 +1,34 @@
+/* Resets the box size of every element */
```

Comment:
Why do we want to do this?

Reply:
It's a reset style sheet. That's its main function.